packages/stream/solana/StreamClient.ts

```diff
@@ -249,7 +251,7 @@ export default class SolanaStreamClient extends BaseStreamClient {
           feeOracle: FEE_ORACLE_PUBLIC_KEY,
           rent: SYSVAR_RENT_PUBKEY,
           timelockProgram: this.programId,
-          tokenProgram: TOKEN_PROGRAM_ID,
+          tokenProgram: programId,
           associatedTokenProgram: ASSOCIATED_TOKEN_PROGRAM_ID,
```

Comment:
I see this is also imported from spl package. It is the same for both token programs?